src/v/cloud_storage/remote_segment.cc

```diff
@@ -1390,10 +1403,16 @@ ss::future<> remote_segment_batch_reader::stop() {
         co_return;
     }
 
-    vlog(_ctxlog.debug, "remote_segment_batch_reader::stop");
+    vlog(
+      _ctxlog.debug,
+      "remote_segment_batch_reader::stop: {}",
```

Comment:
minor nit, to have the same structure of the other logs
```suggestion
      "[{}] remote_segment_batch_reader::stop",
```

Reply:
good point, fixed both